src/components/Animator/components/CSSTransitionWrapper.js

```diff
@@ -57,12 +61,18 @@ class CSSTransitionWrapper extends React.Component {
       case 'entering':
         update = {enter: true, entering: true};
         break;
+      case 'entered':
+        update = {enter: true, entering: true, entered: true};
+        break;
```

Comment:
I looked through the code and I cannot find any usage of entered and exited states. I think we don't need `case 'entered'` and `case 'exited'`